README.md

```diff
@@ -119,6 +119,62 @@ ClientConfig = [{reconnect_cool_down_seconds, 10}],
 ok = brod:start_client([{"localhost", 9092}], brod_client_1, ClientConfig).
 ```
 
+## Handling of connection errors
+
+Usually you would have a supervisor that starts your worker module. In the worker's `start_link` method, you should do two things:
+
+1. Make sure the Brod client you're going to use is running by calling `brod:start_client`. That will start the client under Brod's supervisor, if a client with the given name does not already exist. Afterwards, Brod will automatically recover from connection issues.
+1. Start, e.g., a group subscriber by calling `brod:start_link_group_subscriber`, and return the pid for your supervisor to monitor.
+
+(Note that using `brod:start_link_client` to link the client to your supervisor is deprecated. Using it, a connection loss won't trigger a reconnect, even though the process linked to stays alive.)
```

Comment:
Hm. When using `start_link_client`, I observed one new process below my supervisor, with three children attached to it. When killing Kafka, one of those children died with it and didn't come back; consequently `start_link_group_subscriber` kept failing (I've let it retry every 10 seconds for a few minutes). When using `start_client` the client reconnected after restarting Kafka as expected, so I've assumed differences, but to be honest I didn't go through the code.